qp_target_gene/split_libraries/util.py

```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
     # Get the artifact metadata
     prep_info = qclient.get('/qiita_db/prep_template/%s/'
                             % artifact_info['prep_information'][0])
-    qiime_map = prep_info['qiime-map']
+
+    df = pd.read_csv(
+        prep_info['prep-file'], sep='\t', index_col=0, dtype='str')
+
+    rename_cols = {
+        'barcode': 'BarcodeSequence',
+        'primer': 'LinkerPrimerSequence',
+    }
+    sort_columns = ['BarcodeSequence', 'LinkerPrimerSequence']
+
+    if 'reverselinkerprimer' in df.columns:
+        rename_cols['reverselinkerprimer'] = 'ReverseLinkerPrimer'
+        sort_columns.append('ReverseLinkerPrimer')
+
+    df.rename(columns=rename_cols, inplace=True)
+    # by design the prep info file doesn't have a Description column so we can
+    # prefil without checking
+    index = df.index
+    df['Description'] = pd.Series(['XXQIITAXX'] * len(index), index=index)
```

Comment:
This should work to set this value on all rows:

```suggestion
    df['Description'] = 'XXQIITAXX'
```

Reply:
K